src/setExchangeStatus.ts

```diff
@@ -0,0 +1,37 @@
+import { db, ExchangeStatuses, ExchangeStatus, Schema, Exchange } from './db';
+
+export async function setExchangeStatus(index: number, status: string) {
```

Comment:
Both `commands/setExchangeStatus.ts` and the api should use the same function to update the status. Please refactor the code to get rid of the duplicated logic.

Reply:
I've had the same thoughts but didn't know how to approach it initially. Let me know if the way I did it know would be ok?